agents_infra/agents_infra/agents/base.py

```diff
@@ -156,6 +149,36 @@ def setup_logging(self, log_path=None):
             },
         )
 
+    def aggregate_reports(self, category=None):
+        """
+        Aggregate reports for a given category.
+
+        Parameters:
+            category (str, optional): Report category.
+        """
+        reports = {'server_name': self.server_name}
```

Comment:
For consistency and future extension, consider renaming `reports` to `report_data` or `aggregated_report`.